_build/templates/default/sass/index.scss

```diff
@@ -1481,13 +1481,13 @@ iframe[classname="x-hidden"] {
 }
 
 /* New Header */
-#modx-resource-breadcrumbs {
+.modx-header-breadcrumbs {
   .breadcrumbs {
     display: flex;
     flex-wrap: wrap;
     align-items: baseline;
 
-    #modx-resource-header {
+    h2 {
```

Comment:
For h2, need to add `margin: 0 !important`, because `.modx-page-header` adds extra indentation at the top.